main.py

```diff
@@ -1,54 +1,197 @@
+import argparse
+import os
 import warnings
+from os import listdir
+from os.path import isfile, join
+from random import seed
 
-import numpy as np
+import pandas as pd
 from sklearn import datasets
-from sklearn.metrics import accuracy_score, f1_score
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import StratifiedKFold
 from sklearn.naive_bayes import GaussianNB as Naive
-from sklearn.tree import DecisionTreeClassifier as Tree
-from sklearn.neighbors import KNeighborsClassifier as KNN
 
-from src.ensemble import Ensemble
-from src.flexcon import FlexConC
+import src.utils as ut
+from src.ssl.ensemble import Ensemble
+from src.ssl.self_flexcon import SelfFlexCon
+
+
+crs = [0.05]
+thresholds = [0.95]
 
 warnings.simplefilter("ignore")
-# ssl = FlexConC(Naive(), verbose=True)
 
-# rng = np.random.RandomState(42)
-# iris = datasets.load_iris()
-# random_unlabeled_points = rng.rand(iris.target.shape[0]) < 0.9
-# iris.target_unlabelled = iris.target.copy()
-# iris.target_unlabelled[random_unlabeled_points] = -1
+parser = argparse.ArgumentParser(description="Escolha um classificador para criar um cômite")
+parser.add_argument('classifier', metavar='c', type=int, help='Escolha um classificador para criar um cômite. Opções: 1 - Naive Bayes, 2 - Tree Decision, 3 - Knn, 4 - Heterogeneous')
+parent_dir = "path_for_results"
+datasets_dir = "../FlexConC/datasets"
+datasets = sorted(os.listdir(datasets_dir))
+init_labelled = [0.03, 0.05, 0.08, 0.1, 0.13, 0.15, 0.18, 0.2, 0.23, 0.25]
```

Comment:
Num bagunça, deixa explícito e padronizado em relação aos outros
```suggestion
init_labelled = [0.03, 0.05, 0.08, 0.10, 0.13, 0.15, 0.18, 0.20, 0.23, 0.25]
```

Reply:
feito